.github/workflows/concrete_cpu_test.yml

```diff
@@ -1,17 +1,28 @@
-name: Concrete CPU - Tests
+name: concrete-cpu test
 
 on:
-  workflow_call:
   workflow_dispatch:
+  pull_request:
+      paths:
+        - .github/workflows/concrete_cpu_test.yml
+        - backends/concrete-cpu/**
+  push:
+    branches:
+      - 'main'
+      - 'release/*'
 
 concurrency:
-  group: concrete_cpu_test-${{ github.ref }}
+  group: concrete_cpu_test_${{ github.ref }}
   cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
 
 env:
   CARGO_TERM_COLOR: always
 jobs:
   tests-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        runson: ["ubuntu-20.04"]
```

Comment:
Only one version of the distribution? Why? Is this a dependencies support issue?

Reply:
Just because we not need other distributions for those test, I let the matrix in case we want to add more